From 877f93bcd469b939ec044539b9ea4a0d33e9f177 Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Tue, 11 Mar 2014 15:27:31 +0000
Subject: [PATCH 28/72] Fix remaining use of obsolete 'get_cmd_dir()' in
 plugins

The get_cmd_dir() method was renamed to get_cmd_path(). Fix the
two remaining uses in yum and cluster plugins.

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/cluster.py | 2 +-
 sos/plugins/yum.py     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sos/plugins/cluster.py b/sos/plugins/cluster.py
index 8d73dc1..7957498 100644
--- a/sos/plugins/cluster.py
+++ b/sos/plugins/cluster.py
@@ -97,7 +97,7 @@ class Cluster(Plugin, RedHatPlugin):
                     "crm_from parameter '%s' is not a valid date: using default"
                             % self.get_option('crm_from'))
 
-        crm_dest = os.path.join(self.get_cmd_dir(), 'crm_report')
+        crm_dest = os.path.join(self.get_cmd_path(), 'crm_report')
         self.add_cmd_output('crm_report -S -d --dest %s --from "%s"'
                     % (crm_dest, crm_from))
 
diff --git a/sos/plugins/yum.py b/sos/plugins/yum.py
index 81788f1..aa8cb18 100644
--- a/sos/plugins/yum.py
+++ b/sos/plugins/yum.py
@@ -49,7 +49,7 @@ class Yum(Plugin, RedHatPlugin):
         self.add_cmd_output("subscription-manager list --installed")
         self.add_cmd_output("subscription-manager list --consumed")
         self.add_cmd_output("rhsm-debug system --no-archive --destination %s"
-                % self.get_cmd_dir())
+                % self.get_cmd_path())
 
         if self.get_option("yumlist"):
             # List various information about available packages
-- 
1.9.3

